test/Interpreter/generic_casts.swift

```diff
@@ -139,11 +139,12 @@ enum PE: P {}
 class PC: P {}
 class PCSub: PC {}
 
+// `is` checks
 func nongenericAnyIsP(type: Any.Type) -> Bool {
-  return type is P.Type
+  return type is P.Protocol
```

Comment:
I think `P.Protocol` is actually the correct incantation here since `P` is a protocol identifier and not a generic type variable.